shard-operator-parent/shard-operator/pom.xml

```diff
@@ -23,7 +23,10 @@
       <groupId>com.redhat.service.smartevents</groupId>
       <artifactId>shard-operator-v1</artifactId>
     </dependency>
-
+    <dependency>
```

Comment:
@vaibhavjainwiz We also need to put the `ManagedBridge` CRD into our `kustomize` overlays I think with this change so that it is deployed to the Openshift cluster.

Reply:
done